v0/destinations/indicative/data/INPageConfig.json

```diff
@@ -12,9 +12,10 @@
   {
     "destKey": "eventName",
     "sourceKeys": "name",
```

Comment:
Let's use `["name", "properties.name"]`

For safety

Reply:
done